lib/utils/proxyjump_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"gopkg.in/check.v1"
+)
+
+func (s *UtilsSuite) TestProxyJumpParsing(c *check.C) {
+	type tc struct {
+		in  string
+		out []JumpHost
+		err error
+	}
+	testCases := []tc{
+		{
+			in:  "host:port",
+			out: []JumpHost{{Addr: NetAddr{Addr: "host:port", AddrNetwork: "tcp"}}},
+		},
+		{
+			in:  "host",
+			out: []JumpHost{{Addr: NetAddr{Addr: "host", AddrNetwork: "tcp"}}},
+		},
+		{
+			in:  "bob@host",
+			out: []JumpHost{{Username: "bob", Addr: NetAddr{Addr: "host", AddrNetwork: "tcp"}}},
+		},
+		{
+			in:  "alice@127.0.0.1:7777",
+			out: []JumpHost{{Username: "alice", Addr: NetAddr{Addr: "127.0.0.1:7777", AddrNetwork: "tcp"}}},
+		},
+		{
+			in:  "alice@127.0.0.1:7777, bob@localhost",
```

Comment:
Are we explicitly supporting spaces in jump host lists? All the examples we give have no spaces.

Reply:
yeah, it's fine